<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Search Celeb</string>
    </property>
    <widget class="QDateEdit" name="dateStart">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEnd">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Birthday</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Sex</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxSex">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>JobTitle</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Agency</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxJob">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxAgency">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>75</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>191</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NAME</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BIRTHDAY</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AGE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SEX</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>JOB_TITLE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AGENCY</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
